app/forms/new_phone_form.rb

```diff
@@ -130,6 +132,15 @@ def validate_not_premium_rate
     end
   end
 
+  def validate_cloudfront_matched_automated_request
+    return if cloudfront_matched_automated_request.blank?
+    errors.add(
+      :cloudfront_matched_automated_request,
+      :invalid,
+      message: I18n.t('errors.messages.automated_request'),
+    )
```

Comment:
do we need the `type:` keyword?
```suggestion
    errors.add(
      :cloudfront_matched_automated_request,
      :invalid,
      message: I18n.t('errors.messages.automated_request'),
    )
```

Reply:
I don't think so. I think `type:` is wrong, and I'm on a personal mission to rid it from our codebase 😅 

See https://github.com/18F/identity-idp/pull/9569#issuecomment-1804005889

Reply:
oh no